BiometricAuthentication/app/src/main/AndroidManifest.xml

```diff
@@ -14,7 +14,7 @@
   ~ See the License for the specific language governing permissions and
   ~ limitations under the License
   -->
-<manifest package="com.example.android.fingerprintdialog"
+<manifest package="com.example.android.biomauth"
```

Comment:
nit: Could we use either `bioauth` or `biometricauth` here and elsewhere? Abbreviating it as `biom` seems a bit odd.